invenio_app_rdm/config.py

```diff
@@ -641,6 +641,7 @@ def files_rest_permission_factory(obj, action):
 """Default citation style"""
 
 APP_RDM_DETAIL_SIDE_BAR_TEMPLATES = [
+    "invenio_app_rdm/records/details/side_bar/manageMenu.html",
```

Comment:
In Python we normally use the "underscore" case :)
```suggestion
    "invenio_app_rdm/records/details/side_bar/manage_menu.html",
```